docs/concepts/vega-chain/transactions.md

```diff
@@ -0,0 +1,87 @@
+:::
+
+
+## Validation
+Transactions are validated before they are added to the mempool, and then again once they’re in the mempool but before they are executed. 
+
+A transaction may be rejected throughout the validation process for several reasons, including because it doesn’t pass the spam check, it is malformed or missing information, or conditions have made it irrelevant (for example if a user's balance changes and they no longer have the balance to fulfil an order). 
+
+There are several levels of validation:
+
+1. Vega Wallet ‘sanity check’:
+   If the transaction is sent using the Vega Wallet, the wallet software will ensure that a transaction with a bad payload doesn’t go to the mempool. It checks details including the price formatting, the public key formatting (the right number of characters), the shape of the payload, and more. It also compares fields in the command to ensure they’re not conflicting, such as not specifying fields that are mutually exclusive, or that dependent fields match (such as if the [time in force](../trading-on-vega/orders#times-in-force) is applicable to the order type). 
+   
+   If the wallet validation fails, the transaction is not sent to a node. A transaction that is validated by the wallet, with a success message, is successfully submitted to and ‘ingested’ by a validator node. However, that does not necessarily mean the transaction will be added to a block, which depends on the follow-on validation.
+
+2. Node check: 
+   Once a transaction is sent through a wallet, the validator node it's sent to provides a basic validation check. This check is less thorough than the one performed by Vega Wallet software, but also validates to determine if a transaction should be added to the mempool. A transaction that fails validation does not get added to a block.
+
+3. Validator node consensus check: 
+   Once the transaction is in the mempool, the validator nodes check if the request looks valid, and can be enacted/completed based on the state of the network, and confirms that all the details in the transaction are valid. For example, to vote on a proposal: ensuring the proposal is still live, and that the key has enough tokens to vote. For an order, it confirms, for example, that the assets to cover margin are available, the state of the market is relevant for the order, etc.
+
+::: tip View your transaction
+Once you receive your transaction hash, you can track your transaction’s progress on the [Vega block explorer](https://explorer.fairground.wtf).
+:::
+
+## Filling a block: Transaction gas value
+The proposer of a block takes transactions from their mempool and puts them into a block until the block reaches a maximum total gas amount. This value of <NetworkParameter frontMatter={frontMatter} param="network.transactions.maxgasperblock" hideName={true} /> is set through a network parameter.
```

Comment:
Minor (again it probably just me and not important so feel free to ignore): its not really "set through a network parameter", it _is_ a network parameter. So maybe:
"This value is controlled by the network parameter `network.transactions.maxgasperblock`" or even "The value of the network parameter `network.transactions.maxgasperblock` controls the maximum total gas amount"

Reply:
That is exactly what it does @wwestgarth :) It can also return the name but in this specific case I have it hiding the name and showing the value. But i can still reword it a bit, your point is still reasonable.